src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -397,28 +405,20 @@ private void updateButtonAvailability() {
         boolean isConnected = ooBase != null;
         boolean isConnectedToDocument = isConnected && !ooBase.isDocumentConnectionMissing();
 
-        // For these, we need to watch something
-        boolean hasStyle = true; // (style != null);
-        boolean hasDatabase = true; // !getBaseList().isEmpty();
```

Comment:
These two comments (non-null check / isEmpty) are typical `assert` statements.

Reply:
Okay wait, don't merge yet, I'll see if I can put those conditions (replacing `style`->`currentStyle`) instead of complete removal. I don't know if there were reasons these were commented and true was hardcoded.